packages/piro/src/Piro_NOXSolver_Def.hpp

```diff
@@ -200,11 +210,23 @@ void Piro::NOXSolver<Scalar>::evalModelImpl(
           "Nonlinear solver failed to converge");
     }
 
+    if(appParams->isSublist("Analysis")){
+      Teuchos::ParameterList& analysisParams = appParams->sublist("Analysis");
+      if(analysisParams.isSublist("Optimization Status")) {
+        analysisParams.sublist("Optimization Status").set<bool>("State Solve Converged", solve_status.solveStatus==Thyra::SOLVE_STATUS_CONVERGED);
+        analysisParams.sublist("Optimization Status").set<bool>("Compute State", false);
```

Comment:
You don't need `<bool>` here; the compiler will deduce the type from `false`.

Reply:
OK!